src/secure-proxy-handler.ts

```diff
@@ -206,12 +201,9 @@ export class SecureProxyHandler implements ProxyHandler<SecureProxyTarget> {
         this.initialize(shadowTarget);
         return key in shadowTarget;
     }
-    ownKeys(shadowTarget: SecureShadowTarget): (string | symbol)[] {
+    ownKeys(shadowTarget: SecureShadowTarget): PropertyKey[] {
```

Comment:
⚠️ TypeScript has an incorrect type definition for `Reflect.ownKeys` (https://github.com/microsoft/TypeScript/issues/34626)